spec/support/comment.cr

```diff
@@ -10,3 +8,16 @@ class Comment < Avram::Model
     belongs_to post : Post
   end
 end
+
+class CommentForCustomPost < Avram::Model
+  skip_default_columns
+
+  table comments do
```

Comment:
To keep things [consistent](https://github.com/luckyframework/avram/pull/127), can this be a Symbol? `:comments`

Reply:
Yeah for sure!